aten/src/ATen/native/cpu/Loops.h

```diff
@@ -269,6 +276,8 @@ void unary_kernel(TensorIterator& iter, func_t op) {
     // Specializations to encourage auto-vectorization (trick from Numpy's loops.c.src)
     if (is_unary_contiguous<traits>(strides)) {
       unary_loop(data, strides, 0, n, op);
+    } else if (is_unary_contiguous_s1<traits>(strides)) {
```

Comment:
Plan to take nullary vectorized operations from the #18876 or after it landed or we separate it by two PRs.

Reply:
As I wrote above, it's not the same thing as the "nullary" vectorized operations . This is just a matter of copying these two lines:

```c++
    } else if (is_unary_contiguous_s1<traits>(strides)) {
      unary_loop(data, strides, 0, n, op);
```

in between the `if` and the `else` in `unary_kernel_vec`:

https://github.com/pytorch/pytorch/blob/6e8f528685a12e18e739c720d70aff73f355df68/aten/src/ATen/native/cpu/Loops.h#L297

Please do this as part of this PR because without it there will be cases where `unary_kernel_vec` is slower than `unary_kernel`.

Reply:
Got it.